lib/project_types/script/forms/enable.rb

```diff
@@ -6,18 +6,34 @@ class Enable < ScriptForm
       flag_arguments :api_key, :shop_domain
 
       def ask
-        self.api_key ||= ask_api_key
-        self.shop_domain ||= ask_shop_domain
+        if ScriptProject.current.env.nil?
+          org = ask_org
+          app = ask_app(org)
+          self.api_key = app['apiKey']
+          write_env(org, api_key, app['apiSecretKeys'].first['secret'], nil)
```

Comment:
How come this is different than [forms/push.rb](https://github.com/Shopify/shopify-app-cli/pull/730/files#diff-fd88aa47aeb3c5692bb8b776948176d5R13)?

Reply:
Also are there any plans for DRY? Since there is duplicated code between enable and push

Reply:
Sorry another thing I noticed - do we even need to store the org in the `.env`? I think monorail emits the org_id from the `.shopify-cli.yml` file, so we definitely need to update it there. Should we avoid saving the org in the `.env` then?

And if thats the case.. where is the line drawn between what goes in the `.env` and the `.shopify-cli.yml`?